src/apps/lwaftr/binding_table.lua

```diff
@@ -143,10 +143,31 @@ local BindingTable = {}
 function BindingTable.new(psid_map, br_addresses, softwires)
    local ret = {
       psid_map = assert(psid_map),
-      br_addresses = assert(br_addresses),
-      softwires = assert(softwires)
+      softwires = assert(softwires),
+      br_addresses = cltable.new({key_type = ffi.typeof('uint8_t[16]')})
```

Comment:
I think you're right. I've looked throughout the code and nothing actually uses this table I'm building up. I think initially I built this in when I thought we needed to iterate through the br addresses and then most of subsequently removed it when I realised it wasn't necessary.

I shall remove this and the argument. I also noticed a comment not being quite right.

Thanks for the catch.